### ISSUE TITLE
can't get correct result in run_scGAE.ipynb

### ISSUE THREAD

**Issue by yuxiaokang-source** [ZixiangLuo1161/scGAE#4]
<img width="867" alt="image" src="https://user-images.githubusercontent.com/59059267/156592496-13e08044-f717-42b2-a789-cec164fc92df.png">
<img width="835" alt="image" src="https://user-images.githubusercontent.com/59059267/156592661-b0d68e31-2d10-4793-ba7e-a3fbbae21a48.png">

the `count.npz` seems strange, Is the problem of dataset or the algorithm of scGAE?

**Comment by CisnerosFernandez** [comment]
Hello, 

I am finding the same issue, may I ask if you ever found out its cause?

Thank you, 

Andrés

**Comment by yuxiaokang-source** [comment]
Yes,  I have try many parameters of this network,then I  find the solution of this error .
you should use 
`model = SCGAE(count, adj, adj_n, hidden_dim = 120, latent_dim = 15, decA = "DBL", layer_enc = "TAG")`
not 
`model = SCGAE(count, adj, adj_n, hidden_dim = 120, latent_dim = 15, decA = "DBL", layer_enc = "GAT")`
hope help you

**Comment by CisnerosFernandez** [comment]
Thank you very much
